docs/docsite/rst/shared_snippets/installing_multiple_collections.txt

```diff
@@ -13,7 +13,11 @@ You can also setup a ``requirements.yml`` file to install multiple collections i
      version: 'version range identifiers (default: ``*``)'
      source: 'The Galaxy URL to pull the collection from (default: ``--api-server`` from cmdline)'
 
-The ``version`` key can take in the same range identifier format documented above.
+The support keys for collection requirement entries are ``name``, ``version``, ``source``, and ``type``.
+
+The ``version`` key can take in the same range identifier format documented above. If you're installing a collection from a git repository instead of a built collection artifact, the ``version`` key refers to a git tree-ish.
```

Comment:
Looks like using commit-ish here is a bit more accurate: https://stackoverflow.com/a/23303550/595220.

Also, it's probably worth adding a link to https://git-scm.com/docs/gitglossary#def_commit-ish.

Reply:
TIL. :+1: